### DISCUSSION TITLE
Process community rewards cycle: 12/17/22 - 1/13/23

### DISCUSSION THREAD

**Discussion by AlgoRhythMatic** [asastats/channel#418] [Rewards]
Another cycle has passed, so it is once again time to distribute rewards to our community contributors based on our newly refined measurements listed in the bottom of the page of our [Ongoing Contributions spreadsheet](https://docs.google.com/spreadsheets/d/1UJ9ZyvV0fdcXAfpUNOsRa1aCmJko-QhF-BU4Wcu6Xog/edit#gid=238932857):

```
# NOTE ASA Stats Community rewards 12/17/22 - 1/13/23
@alexander_lakis 0.03 damo
AlgoRhythMatic 0.15 damo
BartleDooDerp  0.27 damo
DK+-x 0.03 damo
kerrilija 0.53 damo
MochaNerd.algo 0.06 damo
motuwagon 0.03 damo
RandomTask 0.06 damo
sealab2022 0.15 damo
t@Miki 0.09 damo
u/AlgoAldo 0.03 damo
u/rockiitp 0.03 damo
TOTAL 1.46 damo
```
Please review and comment if there are any questions or concerns. As note above this period uses newly-refined rewards values discussed here: https://github.com/asastats/channel/discussions/405

Per usual process, we'll wait ~24 hrs for comments/feedback, and then pass on the list to the rewards sender (currently @ipaleka).

As well, please comment or reach out to me if you are interested in participating in rewards measurements in future cycles!

**Comment by ipaleka** [comment]
u/rockiitp gets additional F1 for GARD staking (I mentioned that in comment of the issue, I guess issue body should be the place for that), on top of the one issue for which kerrilija gets IC1.

**Reply by AlgoRhythMatic** [reply]
Wasn't it this comment here:
>As the previous title was misleading (caused by the fact that I forgot that we already have implemented GARD >staking, just not GARDIAN yet), I suggest that two reporters get F1 each:
> u/rockiitp - https://www.reddit.com/r/asastats/comments/109pqwk/new_gardian_staking_on_gardmoney/
> sealab2022 - https://discord.com/channels/906917846754418770/908054304332603402/1062927477938323516

sealab2022 and u/rockiitp each got [F1] from this issue: https://github.com/asastats/channel/issues/415

Maybe it is some other comment elsewhere mentioning u/rockiitp?  I only see this single match when querying for u/rockiitp, and I think it was just singular [F1] value of .03 damo (which is only single reward that u/rockiitp was credited for this cycle AFAIK).

**Reply by ipaleka** [reply]
> Maybe it is some other comment elsewhere mentioning u/rockiitp? I only see this single match when querying for u/rockiitp, and I think it was just singular [F1] value of .03 damo (which is only single reward that u/rockiitp was credited for this cycle AFAIK).

I wasn't able to check last night, it's from this one: https://github.com/asastats/channel/issues/411

So for that one kerrilija gets only IC1, while u/rockiitp gets F1.

**Comment by ipaleka** [comment]
Updated list:

```
# NOTE ASA Stats Community rewards 12/17/22 - 1/13/23
@alexander_lakis 0.03 damo
AlgoRhythMatic 0.15 damo
BartleDooDerp  0.27 damo
DK+-x 0.03 damo
kerrilija 0.47 damo
MochaNerd.algo 0.09 damo
motuwagon 0.03 damo
RandomTask 0.06 damo
sealab2022 0.15 damo
t@Miki 0.09 damo
u/AlgoAldo 0.03 damo
u/rockiitp 0.06 damo
TOTAL 1.46 damo
```

**Reply by AlgoRhythMatic** [reply]
Thank you for the update!  This flight has free high-speed wireless, so came back to check in!

**Reply by ipaleka** [reply]
> Thank you for the update! This flight has free high-speed wireless, so came back to check in!

Thank you mate for thinking about us even on holiday!

**Reply by AlgoRhythMatic** [reply]
I watched a movie on first flight.  This flight is a nice window to catch up a bit!

**Comment by RanTask** [comment]
Thank you guys!
